src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -63,12 +63,6 @@ const GLOBAL_CONFIG_ROOT_ALL_FLAG = `
 --config_root <string>                                    (optional)                        Use configuration files path (default config.NSFS_NC_DEFAULT_CONF_DIR)
 `;
 
-const GLOBAL_CONFIG_FLAGS_ADD_UPDATE_FLAGS = `
```

Comment:
why removing it?

Reply:
I was afraid it might be confusing.
I removed the 'from-file' from this, so I copied those flags in the add and update commands.

Reply:
aha I understand, maybe just bring back the global properties so it will be re-used?

Reply:
I will bring back the global flags including: `config_root`, and `config_root_backend` (without `from-file`).